src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/RSAWrapper.cs

```diff
@@ -215,8 +215,6 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
-        public override bool Equals(object? obj) => _wrapped.Equals(obj);
-
```

Comment:
Should we get rid of `GetHashCode` too?

Reply:
We could. It probably doesn't matter if it is there or not. I generally err on the side of "don't change it without reason".

Reply:
> don't change it without reason

But the result is that the default implementation of `GetHashCode` is changed without reason.

Reply:
> f GetHashCode is changed without reason

I meant "The code has been that way for years and there is no evidence we need to change it."

But since two people seem to feel it should be done, and I don't feel strongly one way or another, sure, why not.